components/Layout/Images/BlurhashImage.vue

```diff
@@ -54,93 +66,132 @@ export default Vue.extend({
     type: {
       type: String as () => ImageType,
       default: ImageType.Primary
+    },
+    lazy: {
+      type: Boolean,
+      default: () => {
+        return true;
+      }
     }
   },
   data() {
     return {
-      image: ''
+      image: '',
+      loading: true
     };
   },
   mounted(): void {
     const img = this.$refs.img as HTMLElement;
     // We don't pass the item itself as we already did all the tags checking in this component,
     // so doing it again in the mixin is useless.
     this.image = this.getImageUrlForElement(this.type, {
-      itemId: this.item.Id,
+      itemId: this.item.Id as string,
       element: img
     });
   },
   methods: {
+    isPerson(obj: BaseItemDto | BaseItemPerson): obj is BaseItemPerson {
+      if ((obj as BaseItemPerson).Role) {
+        return true;
+      }
+      return false;
```

Comment:
Would using `typeof` work here?

Reply:
No to my knowledge, see [here](https://rangle.io/blog/how-to-use-typescript-type-guards/) and [here](https://www.typescriptlang.org/docs/handbook/advanced-types.html)